data/Harvey_2017/metadata.yml

```diff
@@ -0,0 +1,172 @@
+source:
+  primary:
+    key: Harvey_2017
+    bibtype: Article
+    year: '2017'
+    author: J. M. Harvey and A. J. M. Hopkins and M. A. Langley and C. R. Gosper and
+      M. R. Williams and C. J. Yates
+    journal: Australian Journal of Botany
+    title: Long-term studies of post-fire reproduction in an Australian shrubland
+      and woodland
+    volume: '65'
+    number: '4'
+    pages: '339'
+    doi: 10.1071/bt17011
+contributors:
+  data_collectors:
+  - last_name: Gosper
+    given_name: Carl
+    ORCID: 0000-0002-0962-5117
+    affiliation: Biodiversity and Conservation Science, Department of Biodiversity,
+      Conservation and Attractions, WA, Australia
+    additional_role: contact
+  - last_name: Harvey
+    given_name: Judith
+    ORCID: .na
+    affiliation: Biodiversity and Conservation Science, Department of Biodiversity,
+      Conservation and Attractions, WA, Australia
+  austraits_curators: Elizabeth Wenk
+dataset:
+  data_is_long_format: no
+  custom_R_code: '
+    data %>%
+      mutate(Comeditive_stratum = ifelse(PFT_Gosper2012 == "Ephem","post_fire_ephemeral",Comeditive_stratum))
+  '
+  collection_date: 1985/2015
+  taxon_name: Species name
+  location_name: location
+  description: An investigation of patterns of reproduction in south-western Australian
+    kwongan and woodland after prescribed fires with a 30-year longitudinal study,
+    with the aim of identifying appropriate fire intervals.
+  basis_of_record: field
+  life_stage: adult
+  sampling_strategy: Experimental fires ^p Two areas of kwongan and mixed Allocasuarina
```

Comment:
A few headings and new lines (denoted with ^p) left in here

Reply:
Those are intentional - I place them in there as a way to indicate the various paragraphs I've merged together, since we can't add line breaks. (But they also aren't essential - it is just a style I've adopted.)